ledger/src/blockstore.rs

```diff
@@ -262,6 +263,7 @@ pub struct Blockstore {
     perf_samples_cf: LedgerColumn<cf::PerfSamples>,
     rewards_cf: LedgerColumn<cf::Rewards>,
     roots_cf: LedgerColumn<cf::Root>,
+    slot_certificates_cf: LedgerColumn<cf::SlotCertificates>,
```

Comment:
we can rework to remove this before upstreaming to master, it's not needed (an artifact from when we wanted certificate repair)

Reply:
https://github.com/anza-xyz/alpenglow/issues/299